style2.css

```
@@ -0,0 +1,75 @@
+
+body {
+    font-family: Roboto, Halvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    text-transform: uppercase;
+}
+
+.container {
+    
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    text-align: center;
+    position: absolute;
+    width: 1200px;
+    height: 60px;
+    left: 0px;
+    top: 0px;
+    list-style: none;
+
+    background: #FFFFFF;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
+    
+    
+}
+
+.box_1, .box_2,.box_3,.box_4,.box_5,.box_6,.box_7,.box_8 {
+   display: flex;
+   margin: 10px;
+   padding-top: 20px;
+   padding-bottom: 20px;
+}
+
+.box_9 {
+    width: 40px;
+}
+
+.photo {
+    position: absolute;
```

```suggestion
.photo {
  position: absolute;
```
Два отступа для вложенных блоков